resources/views/livewire/order/public-link.blade.php

```diff
@@ -0,0 +1,45 @@
+<div>
+    {{ $order->print()->renderView(FluxErp\View\Printing\Order\Invoice::class) }}
+    <div x-data="signature($wire,$refs)" class="bg-gray-100 pt-10 pr-10">
+        <div class="flex flex-col items-end">
+            <div class="flex w-full justify-between">
+                <div class="flex-1 flex items-center justify-center">
+                    <div x-cloak class="flex flex-col items-center justify-center" x-show="error || id">
+                        <div class="w-10 h-10">
+                            <template x-if="error">
+                                <x-icon name="exclamation"/>
+                            </template>
+                            <template x-if="id && !error">
```

Comment:
why not x-show?

Reply:
if both are hidden(x-show) - same position of message can not be kept when one of them is displayed